core/ts/Poll.ts

```diff
@@ -112,42 +114,36 @@ export class Poll implements IPoll {
    * @param stateTreeDepth - The depth of the state tree.
```

Comment:
The `stateTreeDepth` is no longer a parameter of constructor now, so it should be removed